lib/api/v3/queries/schemas/query_schema_representer.rb

```diff
@@ -105,6 +105,13 @@ def self.filters_schema
                  has_default: true,
                  visibility: false
 
+          schema :timeline_zoom_level,
+                 type: 'String',
+                 required: false,
+                 writable: true,
+                 has_default: true,
+                 visibility: false
```

Comment:
Do we really want to process and send these fixed strings at all times? I don't see the value in doing that so I avoided it. Technically, it would require me to create a special `value_representer` for those, right, or do we have something of the kind `schema_with_allowed_collection_getter` with a fixed array?